src/app/core/header/header.component.html

```diff
@@ -0,0 +1,74 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="container-fluid">
+    <a class="ms-3 navbar-brand" href="#">
+      <img
+        src="../../../assets/zippyfullicon.png"
+        alt="Zippy Icon"
+        height="60px"
+        width="180px"
+      />
+    </a>
+    <button
+      class="navbar-toggler"
+      type="button"
+      data-bs-toggle="collapse"
+      data-bs-target="#navbarNav"
+      aria-controls="navbarNav"
+      aria-expanded="false"
+      aria-label="Toggle navigation"
+    >
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav w-100 d-flex justify-content-end">
+        <li class="mx-2 nav-item">
+          <a class="link nav-link d-flex align-items-center" href="#">
+            <mat-icon class="mx-1">search</mat-icon>
+            Search</a
+          >
+        </li>
+        <li class="mx-2 nav-item">
+          <a class="nav-link d-flex align-items-center" href="#">
+            <mat-icon class="mx-1">local_offer</mat-icon>
+            Offer</a
+          >
+        </li>
+        <li class="mx-2 nav-item">
+          <a class="nav-link d-flex align-items-center">
+            <mat-icon class="mx-1">person</mat-icon>
+            Signin</a
+          >
+        </li>
+        <li class="mx-2 nav-item">
+          <a class="nav-link d-flex align-items-center">
+            <mat-icon class="mx-1">local_mall</mat-icon>
+            Cart</a
+          >
+        </li>
+      </ul>
+      <ul class="navbar-nav d-flex justify-content-end">
+        <li class="mx-2 nav-item dropdown">
+          <a
+            href="#"
+            class="nav-link d-flex align-items-center"
+            data-bs-toggle="dropdown"
+          >
+            <mat-icon class="mx-1 dropdownicon">expand_more</mat-icon>
+            Admin</a
```

Comment:
Replace 'Admin' with 'More'. The dropdown should be open on hover instead of click.